tests/baselines/reference/ambientDeclarations.js

```diff
@@ -77,7 +77,19 @@ declare module 'external1' {
 
 
 //// [ambientDeclarations.js]
+// Ambient variable with type annotation
+// Ambient function with no type annotations
+// Ambient function with type annotations
+// Ambient function with valid overloads
+// Ambient function with optional parameters
+// Ambient class
+// Ambient enum
+// Ambient enum with integer literal initializer
+// Ambient enum members are always exported with or without export keyword

```

Comment:
This output is undesirable because it's extremely misleading. There's a reason we don't emit *every* comment, and this is the typical example of why.